lighthouse-core/config/default-config.js

```diff
@@ -179,6 +179,7 @@ const defaultConfig = {
     gatherers: [
       'http-redirect',
     ],
+    skipPageLoad: true,
```

Comment:
What would be the implications of adding this to `offlinePass`?

Reply:
I don't think we even need offline pass at all anymore, FR gatherer conversion sort of revealed that

Reply:
how do you mean? Is this related to what @patrickhulce and @adrianaixba worked out wrt `service-worker` gatherer?